### ISSUE TITLE
Falta un If comprobando que Model.Enrollments es distinto de null en el Index.cshtml de la vista Intructors

### ISSUE THREAD

**Issue by Corcoleison** [dotnet/AspNetCore.Docs#21720]
https://docs.microsoft.com/es-es/aspnet/core/data/ef-mvc/read-related-data?view=aspnetcore-5.0#modify-the-instructor-index-view

En esta parte, exactamente donde se dice "Después del bloque de código que se acaba de agregar, agregue el código siguiente. Esto muestra una lista de los estudiantes que están inscritos en un curso cuando se selecciona ese curso." 

Nos piden que incluyamos esto:

In this part, exactly where it says "After the code block that was just added, add the following code. This displays a list of the students who are enrolled in a course when that course is selected."

They ask us to include this:
```

<h3>
        Students Enrolled in Selected Course
    </h3>
    <table class="table">
        <tr>
            <th>Name</th>
            <th>Grade</th>
        </tr>
        @foreach (var item in Model.Enrollments)
        {
            <tr>
                <td>
                    @item.Student.FullName
                </td>
                <td>
                    @Html.DisplayFor(modelItem => item.Grade)
                </td>
            </tr>
        }
    </table>
}
```

Pero eso da error, nos falta una condición IF comprobando que Model.Enrollments es distinto de null:
But that gives an error, we are missing an IF condition checking that Model.Enrollments is different from null:

   ```
 @if (Model.Enrollments != null)
    {
        <h3>
            Students Enrolled in Selected Course
        </h3>
        <table class="table">
            <tr>
                <th>Name</th>
                <th>Grade</th>
            </tr>
            @foreach (var item in Model.Enrollments)
            {
                <tr>
                    <td>
                        @item.Student.FullName
                    </td>
                    <td>
                        @Html.DisplayFor(modelItem => item.Grade)
                    </td>
                </tr>
            }
        </table>
    }
}

```

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: f988f508-f5be-9961-830d-aa569adfcce9
* Version Independent ID: 823ce300-dcb9-b51d-fe4b-5bb4625a8126
* Content: [Tutorial: Read related data - ASP.NET MVC with EF Core](https://docs.microsoft.com/en-us/aspnet/core/data/ef-mvc/read-related-data?view=aspnetcore-5.0)
* Content Source: [aspnetcore/data/ef-mvc/read-related-data.md](https://github.com/dotnet/AspNetCore.Docs/blob/master/aspnetcore/data/ef-mvc/read-related-data.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-data**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

---
#### Detalles del documento

⚠ *No edite esta sección. Se requiere para docs.microsoft.com ➟ Vinculación de problema de GitHub.*

* ID: f988f508-f5be-9961-830d-aa569adfcce9
* Version Independent ID: 823ce300-dcb9-b51d-fe4b-5bb4625a8126
* Content: [Tutorial: Lectura de datos relacionados: ASP.NET MVC con EF Core](https://docs.microsoft.com/es-es/aspnet/core/data/ef-mvc/read-related-data?view=aspnetcore-5.0)
* Content Source: [aspnetcore/data/ef-mvc/read-related-data.md](https://github.com/dotnet/AspNetCore.Docs.es-es/blob/live/aspnetcore/data/ef-mvc/read-related-data.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-data**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

**Comment by Rick-Anderson** [comment]
@Corcoleison  good catch. I'll fix this in  #21576

**Comment by Corcoleison** [comment]
> @Corcoleison good catch. I'll fix this in #21576

Thank you for your work!

**Comment by Rick-Anderson** [comment]
This is fixed in RP version but not MVC version.
